aip/0132.md

```diff
@@ -78,17 +79,17 @@ message ListBooksRequest {
 - The `page_size` and `page_token` fields, which support pagination, **must**
   be included on all list request messages. For more information, see
   [AIP-158][].
-  - The comment above the message **should** document the maximum allowed
-    value, as well as the default value if the field is omitted (or set to
-    `0`). If preferred, the API **may** state that the server will use a
-    sensible default that may vary over time.
+  - The comment above the `page_size` field **should** document the maximum
+    allowed value, as well as the default value if the field is omitted (or set
+    to `0`). If preferred, the API **may** state that the server will use a
+    sensible default. This default **may** change over time.
```

Comment:
Can the maximum allowed value change over time too? Presumably only upwards.

Reply:
I think it should be able to change in either direction. We already state that we might send fewer than you ask for and generally have enough restrictions that the only thing a user can do is loop over the results and check for a next page token. Any code that someone may have written that checks an index (`results[24]`) or does anything meaningful based on the number of results was _already_ wrong and could break for other reasons.

Reply:
I think it's okay *only* because if asked for more than the server is willing to return, it truncates rather than erroring out.